app/services/translation-map.js

```diff
@@ -1,6 +1,2 @@
 import Ember from 'ember';
-
-let map = {
-  __TRANSLATION_MAP__: {}
-};
-export default Ember.Service.extend({ map: map.__TRANSLATION_MAP__ });
```

Comment:
the new version of ember-cli-uglify was smart enough to see that `map.__TRANSLATION_MAP__` is an empty object and outputed

```javascript
export default Ember.Service.extend({ map: {}});
```